Resources/Textures/Objects/Consumable/Drinks/hot_moffee.rsi/meta.json

```
@@ -0,0 +1,28 @@
+{
+	"version": 1,
+	"size": {
+		"x": 32,
+		"y": 32
+	},
+	"license": "CC-BY-SA-3.0",
+	"copyright": "Original Coffee sprite from https://github.com/discordia-space/CEV-Eris/tree/master/icons/obj/drinks.dmi at commit 385fd6dcbd57498109f5758b4dc0ea2021ba0af8, recolored by OrnamentalFreak (GitHub)",
```

I may have forgotten to clarify that it was hot_coffee, as that's the sprite you used, right?

Also, I linked directly to the latest git commit on GitHub that changed anything in drinks.dmi.

```suggestion
	"copyright": "Original hot_coffee sprite from https://github.com/discordia-space/CEV-Eris/tree/385fd6dcbd57498109f5758b4dc0ea2021ba0af8/icons/obj/drinks.dmi at commit 385fd6dcbd57498109f5758b4dc0ea2021ba0af8, recolored by OrnamentalFreak (GitHub)",
```

---

Ah, we _both_ missed that. Thanks for taking the second swing at it, that is in fact correct.